tests/test_package.py

```diff
@@ -191,5 +192,58 @@ def test_list_commands() -> None:
         output = subprocess.check_output(
             ["dbos", "workflow", "queue", "list"], cwd=temp_path
         )
-        data = json.loads(output)
-        assert isinstance(data, list) and len(data) == 10
+        workflows = json.loads(output)
```

Comment:
This is really good, much of it was missing before. It all uses `dbos-config.yaml` though as far as I can tell.

Reply:
Good point I'll add the `--db-url` option